.github/scripts/workflow-commands.functions.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
the name implies the file is only meant for GH workflow commands or any SH commands can be used?

Reply:
It’s for GitHub API interface only.

Reply:
IMHO `logging.function.sh` would be more obvious